var/spack/repos/builtin/packages/exago/package.py

```diff
@@ -160,7 +161,7 @@ class Exago(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("petsc@3.13:3.14", when="@:1.2")
     depends_on("petsc@3.16", when="@1.3:1.4")
     depends_on("petsc@3.18:3.19", when="@1.5")
-    depends_on("petsc@3.20:", when="@1.6:")
+    depends_on("petsc@3.19:", when="@1.6:")
```

Comment:
@balay added this. I think it's safer just to use 3.20+ and move on from 3.19

Reply:
Hm - looking at my prior change at dbf21bf843dd81721dd0affc3a5c368d8099ff38

Looks like petsc@3.20 was not in the dependency list - so my change added it in.

If petsc@3.19 works with @1.6 - its fine to add in this additional change.